### PR TITLE
Insert Scraped Data into Postgres

### PR DESCRIPTION
**Pull request by gamburgm**
Bulk insert (or update) scraped data into Postgres.

### CODE REVIEW

File: backend/dumpProcessor.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import _ from 'lodash';
+import Keys from '../common/Keys';
+import macros from './macros';
+import db from './database/models/index';
+
+const Professor = db.Professor;
+const Course = db.Course;
+const Section = db.Section;
+
+class dumpProcessor {
```
Comment (gamburgm) [inline backend/dumpProcessor.js]: Not sure if specs are necessary here—we could spec that the data is getting processed in the process functions and then that `bulkCreate` does what we expect it to, but neither of those things seem worth spec-ing out.

File: backend/dumpProcessor.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    const additionalProps = { id: `${Keys.getClassHash(classInfo)}`, minCredits: Math.floor(classInfo.minCredits), maxCredits: Math.floor(classInfo.maxCredits) };
+    return { ...classInfo, ...additionalProps };
+  }
+
+  processSection(secInfo) {
+    const additionalProps = { id: `${Keys.getSectionHash(secInfo)}`, classHash: Keys.getClassHash(secInfo) };
+    return { ...secInfo, ...additionalProps };
+  }
+
+  chunkify(arr) {
```
Comment (dajinchu) [inline backend/dumpProcessor.js]: https://lodash.com/docs/4.17.15#chunk

File: backend/tests/dumpProcessor.test.js [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+
+it('does not create records if dump is empty', async () => {
+  const prevCounts = await getCounts();
+  await dumpProcessor.main({ classes: [], sections: [] }, {});
+  expect(await getCounts()).toEqual(prevCounts);
+});
+
+describe('with professors', () => {
+  it('creates professors', async () => {
+    const prevCounts = await getCounts();
```
Comment (dajinchu) [inline backend/tests/dumpProcessor.test.js]: are we unable to just clean out the whole table before each tests?

**Review by dajinchu — CHANGES_REQUESTED** [review]
i see we deleted meetings. do we need a migration to drop the meetings table then?

File: backend/dumpProcessor.js (hunk 2 above)
Comment (gamburgm) [reply backend/dumpProcessor.js]: Nice

File: backend/tests/dumpProcessor.test.js (hunk 3 above)
Comment (gamburgm) [reply backend/tests/dumpProcessor.test.js]: I was already doing that, not sure what I was thinking

**Review by gamburgm — COMMENTED** [review]
> i see we deleted meetings. do we need a migration to drop the meetings table then?

I was honestly thinking of just dropping the database and re-building it on these migrations, I didn't want to pollute our migrations with the changes we made as we figure out the best way to configure our data early on.

**Review by dajinchu — APPROVED** [review]
👍

**Pull request merged**